docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+* ``flagsoldout``: is the conference sold out?
+* ``flagcfp``: is the CFP currently open?
+* ``flagspeakersannounced``: have speakers been announced?
+  This flag requires the ``_data/<shortcode>-<year>-sessions.yaml``
+  file to exist, which lists the speakers/sessions.
+* ``flaghasschedule``: is the schedule ready?
+  This flag requires the``_data/<shortcode>-<year>-schedule.yaml``
+  file to exist, which contains the schedule.
+* ``flaglivestreaming``: is a live stream currently running?
+* ``flagpostconf``: has the conference ended?
+* ``flagvideos``: are the conference videos published?
+
+
+Sessions file
+-------------
+
+The sessions file contains the conference sessions, i.e. the talks.
+
+Each talk has the following attributes:
+
+* ``title``: title of the talk
+* ``abstract``: talk abstract
+* ``slug``: talk slug - referenced in the schedule.Typically, the slug is
+  a slugified version of the title, followed by the slugified speaker name.
+* ``series``: the conference series, used for videos, e.g. ``Write the Docs PORTLAND``
+* ``series_slug``: the slug of the series, used for videos
+* ``year``: the year the talk was given, used for videos
+* ``youtubeId``: the Youtube ID of the talk video, if published already
+* ``speakers``: the speaker(s) for the talk, in a list of speakers with keys:
+    * ``name``: name of the speaker
+    * ``slug``: slug of the speaker
+    * ``twitter``: Twitter username
+    * ``website``: URL of the speaker's website
+
+
+Schedule file
+-------------
+
+The schedule file contains the conference schedule. This is a mix of conference
+talks, and other agenda items like "Switch Speakers" or "Lunch Break".
+
+If you're writing a new conference schedule, it may be easier to start from
+the schedule file of the same conference as last year, as they tend to be
+quite similar.
+
+For conferences with a writing day, there must be a ``writing_day`` key. Then,
+a schedule for each main conference day, in the form of ``day1``, ``day2``,
+and more if needed. The number of days should match ``date.total_talk_days``
+from the general config file.
+
+Within each day, each item must have a ``time``, which is free text, so it
+can be in 12 hour or 24 hour time, and either a ``title`` or a ``slug``.
+A title is used for free text schedule items, like "Snack Break". A slug is
+used for conference talks, where the slug must match the slug of a session
+in the ``_data/<shortcode>-<year>-sessions.yaml`` file.
+
+A schedule file for a very brief conference could look like::
+
+    writing_day:
+      - time: '8:00'
+        title: Doors Open, Breakfast Served
+      - time: '5:00'
+        title: End of Writing Day
+
+    day1:
+      - time: '8:00'
+        title: Doors Open, Breakfast Served
+      - time: '9:40'
+        title: "<b>Unconference Starts (Lola's Room)</b>"
+      - time: '9:40'
+        slug: any-friend-of-the-docs-is-a-friend-of-mine-cultivating-a-community-of-documentation-advocates-heather-stenson
```

Comment:
is this a copy-paste operation?

Reply:
The slugs are, yes. The file as a whole too, as a start, because we have almost identical schedules each year per conference. Clarified that elsewhere.